Makefile

```diff
@@ -209,6 +213,14 @@ alt_firmware: \
 	check_s2740vc-v1_default \
 	sizes
 
+# builds with RTPS
+check_rtps: \
```

Comment:
Is the plan to add this to CI? If so, we'll need to check the overhead of building all targets.

Reply:
Yes, this is for CI.